File: Ex91_HelloKotlin/app/src/main/java/com/example/ex91_hellokotlin/Practice.java
package com.example.ex91_hellokotlin;

public class Practice {
}


class ParentJava{

    String name;
    int age;

    public ParentJava(int age,String name){
        this.name = name;
        this.age = age;
    }
}

class ChildJava extends ParentJava{

    public ChildJava(int age, String name) {
        super(age, name);
    }
}

File: Ex91_HelloKotlin/app/src/main/java/com/example/ex91_hellokotlin/Practice.kt
package com.example.ex91_hellokotlin

fun main(){
    val parent:Parent = Parent("sam")
    val child:Child = Child("robin",25)
}

open class Parent(var name:String){
    init{
        println("Parent 의 주 생성자 : $name")
    }

}

class Child(name:String, var age:Int) : Parent(name){
    init{
        println("Child 의 주 생성자 : $name , $age")
    }

}